pkg/zenhub/issue_service.go

```diff
@@ -281,14 +290,49 @@ func (s IssueService) SetProgress(issue issues.IssueRef, column string) error {
 	return nil
 }
 
+func (s IssueService) SplitIssueRef(issue issues.IssueRef) (string, string, int, error) {
```

Comment:
Is this duplicating the functionality in `issues.IssueRef.Parts`? Can we use that instead of adding a method to IssueService?